docker/config/local_dev.env

```diff
@@ -36,3 +36,4 @@ ELIOT_LOGGING_LEVEL=INFO
 ELIOT_STATSD_HOST=statsd
 ELIOT_STATSD_NAMESPACE=mcboatface
 ELIOT_STATSD_PORT=8125
+ELIOT_SYMBOLS_CACHE_MAX_SIZE=400000000
```

Comment:
Setting the cache max size to 400,000,000 forces the symbols cache to update regularly. It can hold about 40 files in it.